huaweicloud/common/common.go

```diff
@@ -319,3 +319,20 @@ func RetryContextWithWaitForState(param *RetryContextWithWaitForStateParam) (int
 
 	return stateConf.WaitForStateContext(param.Ctx)
 }
+
+// GetEipsbyAddresses returns the EIPs of addresses when success.
+func GetEipsbyAddresses(client *golangsdk.ServiceClient, addresses []string) ([]eips.PublicIp, error) {
```

Comment:
please func GetEipIDAddress in common.go
the enterprise_project_id need to be take care

Reply:
done